README.md

```diff
@@ -18,31 +18,47 @@ Quick Start
     See [Obtaining OpenTok Credentials](#obtaining-opentok-credentials)
     for important information.
  
- 1. Install CocoaPods as described in [CocoaPods Getting Started](https://guides.cocoapods.org/using/getting-started.html#getting-started).
+ 2. Install CocoaPods as described in [CocoaPods Getting Started](https://guides.cocoapods.org/using/getting-started.html#getting-started).
```

Comment:
usually, numbering is better to simple use `1. ... 1. ...` This is intentionally done so when you change steps, you don't have to do manual task of reordering them.